.github/workflows/ci.yml

```diff
@@ -17,14 +17,8 @@ jobs:
       matrix:
         # os: [ ubuntu-20.04, windows-latest, macOS-latest ]
         os: [ ubuntu-20.04 ]
-        ros_distribution: [ foxy, rolling, dashing ]
```

Comment:
I also don't understand why you removed foxy and dashing. This is the ci-job for the entire rclc repository! So 
- until May 2021 we need a CI job for
   - Dashing (with foxy branch)
   - Foxy (with foxy branch)
   - Rolling (with master branch)
  
- After May 2021 we need a CI job for
  - Foxy (with foxy branch) 
  - Galactic (with galactic branch (this was the previous master branch)
  - Rolling (with master branch)

Reply:
This PR will be merged into master, so after merge the `ci.yml` _of the master branch_ will only build for rolling.
The `ci.yml` of the foxy branch is not influenced by this PR and will/should still build for foxy+dashing.

This is my understanding at least, @JanStaschulat and @pablogs9.

Reply:
OK. understood
- I updated the CI-job for ros2/rclc:foxy branch in this PR https://github.com/ros2/rclc/pull/67
- TODO: update bloom release for Dashing (change from `master`-> `foxy` branch)
Then I guess everything is well again, right?

Reply:
The point is that since now we are going to have separate branches for each ROS2 release, so I'm ok with keeping just `rolling` in master and having https://github.com/ros2/rclc/pull/67#pullrequestreview-621995126. Is that right?